_i18n/zh-cn.yml

```
@@ -16,75 +16,75 @@ footer:
   resources: 资源
-  asset_library: 资产库
+  asset_library: 资源库
   press_kit: 宣传材料
-  showcase: 展示
+  showcase: 案例展示
   education: 教育
   foundation: 基金会
   about: 关于
   donate: 捐助
   license: 许可证
   privacy_policy: 隐私政策
   contact_us: 联系我们
-  text: "Juan Linietsky，Ariel Manzur 与<a href='https://github.com/godotengine/godot/blob/master/AUTHORS.md' target='_blank' rel='noopener'>贡献者们</a>。由 <a href='https://godot.foundation/' target='_blank' rel='noopener'>Godot 基金会</a>架设。网站<a href='https://github.com/godotengine/godot-website' target='_blank' rel='noopener'>源码可在 GitHub 上获取</a>."
+  text: "Juan Linietsky，Ariel Manzur 和<a href='https://github.com/godotengine/godot/blob/master/AUTHORS.md' target='_blank' rel='noopener'>其他贡献者们</a>。由 <a href='https://godot.foundation/' target='_blank' rel='noopener'>Godot 基金会</a>架设。网站<a href='https://github.com/godotengine/godot-website' target='_blank' rel='noopener'>源码可在 GitHub 上获取</a>."
 sponsors:
-  title: "Godot 由以下企业赞助："
-  text: 如果您对企业赞助感兴趣，请发邮件至 <a href='mailto:contact@godot.foundation'>contact@godot.foundation</a>。<br>作为个人捐助，你可以通过 <a href='https://fund.godotengine.org/'>Godot Engine 开发基金</a> 进行。
+  title: "Godot 的赞助者们："
+  text: 如果您对企业赞助感兴趣，请发邮件至 <a href='mailto:contact@godot.foundation'>contact@godot.foundation</a>。<br>以个人名义捐助，您可以通过 <a href='https://fund.godotengine.org/'>Godot Engine 开发基金</a> 进行捐助。
   cta: 捐助
 home:
-  h1: 属于你的免费，<br>开源游戏引擎。
-  subtitle: 开发你的 2D 与 3D 游戏，跨平台项目，甚至是 XR 创意！
-  whats_new: 最近更新
+  h1: 独属于你的，<br>自由开源游戏引擎。
+  subtitle: 开发你的 2D、3D 游戏，跨平台项目，甚至是 XR 创意！
+  whats_new: 新版亮点
   download_latest: 下载最新版
-  looking_for: '正在寻找 <a href="/download/windows/" class="set-os-download-url" data-version="3" title="下载 Godot 3 的长期支持版本">Godot 3</a>，我们的<a href="/download/archive">实验性版本</a>，或是一个<a href="/download/archive">历史版本</a>？'
-  latest_news: 最近新闻
-  more_news: 更多新闻
+  looking_for: '正在寻找 <a href="/download/windows/" class="set-os-download-url" data-version="3" title="下载 Godot 3 的长期支持版本">Godot 3</a>？我们的<a href="/download/archive">测试版本</a>？或是<a href="/download/archive">历史版本</a>？'
+  latest_news: 最新消息
+  more_news: 更多消息
   cards:
-    h2: 制作游戏的另一种方式
+    h2: 打造游戏的独特方式
     design:
       title: 创新设计
-      text: Godot 的节点与场景系统同时给了你随心创造所需的能力与灵活性。
+      text: Godot 的节点和场景系统兼具强大和灵活，可以创造任何事物。
     script:
-      title: 使用适合工作的语言
-      text: 使用 Godot 自带的 GDScript、C#、C++ 等面向对象的 API 保持代码模块化，或通过 GDExtension 引入自定义语言。
-      extra: 截止 Godot 4.2 ，C# 支持目前仅在桌面端与移动端可用。<br>使用 Godot 3 来在网页端运行 C# 。
+      title: 使用适合的语言完成工作
+      text: 保持代码模块化，使用面向对象的 API，您可以使用 Godot 原生的 GDScript, C#, C++, 或是使用GDExtension 引入您自己的语言。
+      extra: 截止 Godot 4.2 ，C# 支持目前仅在桌面端与移动端可用。<br>使用 Godot 3 在网页端运行 C# 。
     2d:
       title: 专用 2D 引擎
-      text: 利用 Godot 专有的 2D 渲染引擎，基于真实的 2D 像素坐标和 2D 节点系统，制作精美且性能卓越的 2D 游戏。
+      text: 使用 Godot 专用的 2D 渲染引擎，基于真实的 2D 像素坐标和 2D 节点，制作精美且性能卓越的 2D 游戏。
     3d:
       title: 简单且强大的 3D
-      text: Godot 的 3D 节点为你提供了构建、动画制作及渲染 3D 世界与角色所需的一切功能。
+      text: Godot 的 3D 节点为你提供了构建、动画制作和渲染 3D 世界与角色所需的一切功能。
```

此处的主语是 3D worlds and characters，动作是 build, animate, and render。

目前的两种翻译方法都很难断句。建议：

```suggestion
      text: 使用 Godot 的 3D 节点即可对 3D 世界和角色进行构建、动画、渲染。
```